tst/com/amazon/corretto/crypto/provider/test/EcGenTest.java

```diff
@@ -205,6 +245,8 @@ public void unknownSize() throws GeneralSecurityException {
                 () -> nativeGen.initialize(13));
     }
 
+    // TODO Update EXPLICIT_CURVE to non-legacy curve
```

Comment:
Let's fix this now. It isn't that hard to do. Just pull the values from here: https://github.com/corretto/corretto-8/blob/e6eb5c6a8c985ff8f1aa299d87fe5b798b5c2a3e/src/jdk/src/share/classes/sun/security/util/CurveDB.java#L298-L305